lib/shrine/plugins/validate_virus_free.rb

```diff
@@ -4,8 +4,14 @@ module Plugins
     module ValidateVirusFree
       module AttacherMethods
         def validate_virus_free(message: nil)
-          cached_path = "#{Rails.root}/#{get.to_io.path}"
-          ClamScan::Client.scan(location: cached_path).safe? || add_error(message) && false
+          if ClamScan.configuration.client_location.blank?
+            Shrine.logger.error('NO VIRUS SCANNING ENABLED')
+            return
+          end
+          cached_path = get.download.path
+          result = ClamScan::Client.scan(location: cached_path)
+          # TODO: Log a the full result to sentry
+          result.safe? || add_error(message) && false
```

Comment:
They get wrapped in an `ArgumentError` for now, and I have some work to put them in a custom `Common::Exception` wrapper in progress elsewhere.